apps/src/javalab/control-buttons.module.scss

```diff
@@ -0,0 +1,53 @@
+@import 'color.scss';
+
+@mixin controlButton {
```

Comment:
I'm looking back at this now, and thinking using extend might be cleaner. Is that right?

Reply:
I must say this question is out of my wheelhouse ... But I usually think of needing mixins when styles need to change based on arguments passed in. So extend seems more in line with what I've seen for these?

Reply:
+1, it's been a while since I did SCSS but I think @extend would be more appropriate here?

Reply:
Thanks both -- I'll look at switching to extend 👍

Reply:
Found this summary helpful:

> Extends and [mixins](https://sass-lang.com/documentation/at-rules/mixin) are both ways of encapsulating and re-using styles in Sass, which naturally raises the question of when to use which one. Mixins are obviously necessary when you need to configure the styles using [arguments](https://sass-lang.com/documentation/at-rules/mixin#arguments), but what if they’re just a chunk of styles?

> As a rule of thumb, extends are the best option when you’re expressing a relationship between semantic classes (or other semantic selectors). Because an element with class .error--serious is an error, it makes sense for it to extend .error. But for non-semantic collections of styles, writing a mixin can avoid cascade headaches and make it easier to configure down the line.